src/CurrencyData.js

```diff
@@ -133,6 +133,13 @@ const CurrencyData = {
         "rounding": 0,
         "code": "BRL"
     },
+    "BTC": {
+      "symbol": "฿",
```

Comment:
While it looks very similar, the Thai Bath is not Bitcoin :-)  I'd rather have no symbol instead and show the three-letter code instead of the wrong symbol. 

It would look weird especially for any exchanges that actually have the THBBTC pair.